### ISSUE TITLE
Error: startWith on undefined variable 'req.ip'

### ISSUE THREAD

**Issue by VM-2020** [Ylianst/MeshCentral#1842]
```
-------- 9/21/2020, 1:58:20 PM ---- 0.6.36 --------

TypeError: Cannot read property 'startsWith' of undefined
    at /var/www/html/............................................../node_modules/meshcentral/webserver.js:4585:34
    at Layer.handle [as handle_request] (/var/www/html/.............................................../node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/var/www/html/............................................................../node_modules/express/lib/router/index.js:317:13)
    at /var/www/html/....................................................../node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/var/www/html/....................................................../node_modules/express/lib/router/index.js:335:12)
    at next (/var/www/html/............................................................./node_modules/express/lib/router/index.js:275:10)
    at _cookieSession (/var/www/html/............................................................/node_modules/cookie-session/index.js:129:5)
    at Layer.handle [as handle_request] (/var/www/html/....................................................../node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/var/www/html/......................................................../node_modules/express/lib/router/index.js:317:13)
    at /var/www/html/......................................................./node_modules/express/lib/router/index.js:284:7
```

**Comment by Ylianst** [comment]
Working on this now!

**Comment by Ylianst** [comment]
Interesting. I just published MeshCentral v0.6.45 with a fix for this, however I am not sure when a situation where an incoming request has to IP address. I will default to "0.0.0.0" in that case instead of a server exception.
